blueocean-rest-impl/pom.xml

```diff
@@ -97,6 +97,7 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
```

Comment:
Removes the following unnecessary files from the JPI:

- `byte-buddy-1.12.13.jar`
- `byte-buddy-agent-1.12.13.jar`
- `mockito-core-4.7.0.jar`
- `objenesis-3.2.jar`